### PR TITLE
[Jasoet] Enable Support for Yaml Config

### PR DESCRIPTION
**Pull request by jasoet**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  write.buffer.size: 4096
+postgres:
+  user: postgres
+  password:
+  host: localhost
+  port: 5432
+  database: proctord_test
+  max.connections: 50
+  connections.max.lifetime: 30
+new.relic:
```
Comment (BimoZX) [inline config.yaml:27]: `new.relic` or `new_relic`? Since the former might be confusing in determining the namespace of the config.

**Review by walbertus — APPROVED** [review]
Mantoel

**Review by BimoZX — COMMENTED** [review]
Might be better to use `_` instead of `.` as a separator.

File: config.yaml (hunk 1 above)
Comment (jasoet) [reply config.yaml:27]: It legacy code will break deployment if changed.

**Review by BimoZX — APPROVED** [review]

**Review by mazbergaz — APPROVED** [review]

**Pull request merged**